stellar/deploy-contract.js

```diff
@@ -96,36 +101,83 @@ async function processCommand(options, config, chain) {
     await broadcast(operation, wallet, chain, 'Initialized contract', options);
 }
 
+async function processUpgrade(options, _, chain) {
+    const { contractName, contractId, privateKey, wasmPath, newWasmHash } = options;
```

Comment:
where is `contractId` coming from? Shouldn't the address be read from the config?